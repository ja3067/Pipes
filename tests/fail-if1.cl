def dynamic():
	if True:
		return 3
	else:
		return 4.0

if dynamic():
	print(3)
