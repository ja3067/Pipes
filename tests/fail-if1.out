RuntimeError: invalid boolean type in if statement
